_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-podcast-player.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Podcast Player API
+ *
+ * @package Jetpack
+ * @since 8.4.0
+ */
+
+/**
+ * Fetch podcast feeds and parse data for the Podcast Player block.
+ *
+ * @since 8.4.0
+ */
+class WPCOM_REST_API_V2_Endpoint_Podcast_Player extends WP_REST_Controller {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'podcast-player';
+		// This endpoint *does not* need to connect directly to Jetpack sites.
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register the route.
+	 */
+	public function register_routes() {
+		// GET /sites/<blog_id>/podcast-player - Returns feed data.
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'  => WP_REST_Server::READABLE,
+					'callback' => array( $this, 'get_player_data' ),
```

Comment:
Should we add a basic user permissions here?

```
if ( ! current_user_can( 'edit_posts' ) ) {
			return new WP_Error(
				'rest_user_cannot_view',
				__( 'Sorry, you are not allowed to access remote urls.', 'jetpack' )
			);
		}
```

Reply:
Follow up created